### PR TITLE
Repair ferrocene.document-signatures option

### PR DESCRIPTION
**Pull request by Hoverbear**
Addresses https://github.com/ferrocene/ferrocene/issues/788

### CODE REVIEW

File: ferrocene/ci/configure.sh [hunk 1]
```diff
@@ -345,7 +345,7 @@ fi
 # access to the document signature files stored in AWS. In that case, configure
 # the build system to ignore document signatures.
 if ! is_internal; then
-    add --set ferrocene.ignore-document-signatures=true
+    add --set ferrocene.document-signatures=disable
```
Comment (tshepang) [inline ferrocene/ci/configure.sh:348]: is this a typo
```suggestion
    add --set ferrocene.document-signatures=disabled
```

Comment (Hoverbear) [reply ferrocene/ci/configure.sh:348]: My danged mouse clicker. Thank you.

**Review by tshepang — APPROVED** [review]
bors merge

**Comment by steev** [comment]
Tested locally and it does generate the config.toml correctly for me :)  Thanks for the quick turnaround!

**Pull request merged**